src/formio/validators/minMaxDateValidator.js

```diff
@@ -8,8 +8,9 @@ const MinMaxDateValidator = {
     const minDate = new Date(component.component.minDate);
     const maxDate = new Date(component.component.maxDate);
 
-    const errorKeys = component?.openForms?.validationErrorContext?.minMaxDateValidatorErrorKeys;
-    const errorMessage = errorKeys ? errorKeys[0] : 'invalidDate';
```

Comment:
formio actually documents the `invalid_date` error key, so there may be a camel case conversion thing in play.

Reply:
I checked this with our custom `invalid_time` thing and I don't see any usage of `invalidDate` (camelcase) in Formio, so looks like it's just a bug and we should fall back to `invalid_date` like Formio itself does